test/solidlsp/scala/test_scala_language_server.py

```diff
@@ -0,0 +1,53 @@
+import logging
+import os
+
+import pytest
+
+from solidlsp.language_servers.scala_language_server import ScalaLanguageServer
+from solidlsp.ls_config import Language, LanguageServerConfig
+from solidlsp.ls_logger import LanguageServerLogger
+from solidlsp.settings import SolidLSPSettings
+
+
+@pytest.fixture(scope="module")
```

Comment:
Please review if there are any test cases that should be added.

Reply:
Since the granularity of tests varies depending on the language implementation, it is difficult to know how much test cases to create and from what perspective.